Lib/test/test_threading.py

```diff
@@ -557,7 +557,7 @@ def exit_handler():
         """)
         _, out, err = assert_python_ok("-c", code)
         self.assertEqual(out, b'')
-        self.assertEqual(err.rstrip(), b'child process ok')
+        self.assertIn(b"can't fork at interpreter shutdown", err)
```

Comment:
This test_fork_at_exit() test doesn't seem to make sense to be in test_threading anymore after this PR's primary change.  you added an equivalent without the misleading no longer true functionality to test_os anyways.